### DISCUSSION TITLE
Tags important to search question...??

### DISCUSSION THREAD

**Discussion by kushshiv** [IndianOpenSourceFoundation/dynamic-cli#135] [General]
Hi All... Why is it mandatory to input `Tags` prompt...?? I guess this should not be mandatory to answer... As people always search for question and not the tags. Can we have it as an optional field...?? 
![image](https://user-images.githubusercontent.com/33164379/142460652-ef73107f-538e-400b-8719-e6be98aa04c9.png)

We can do something like this on constants to check if question is mandatory or not?
```
queries = [
    {"query" : "What do you want to search", "is_mandatory" : True},
    {"query" : "Tags", "is_mandatory" : False}
]
```

**Comment by GouravSardana** [comment]
Hey,

Thanks for participating in discussion. Yes we can mark is as optional but I think by giving tag would make more sense to inclined to the question. What are other views ?
@pranavbaburaj @guptaaastha @NamamiShanker

**Reply by lainq** [reply]
Yeah. i think we can make tags optional

**Comment by lainq** [comment]
Please take a look at #136
